index.html

```diff
@@ -1,25 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Christian Portfolio</title>
     <link rel="stylesheet" href="css/style.css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/ad432cd04f.js" crossorigin="anonymous"></script>
 </head>
+
 <body>
+    <div id="menu-display" class="mobile-toggler menu-display">
+        <div class="mobile-toggler-btn">
+            <i class="fa-solid fa-xmark" id="menu-btn-2"></i>
+        </div>
+        <ul>
+            <li id="work-btn"><a href="#work-section">Portfolio</a></li>
+            <li id="about-btn"><a href="#about-section">About</a></li>
+            <li id="contact-btn"><a href="#contact-section">Contact</a></li>
+        </ul>
+    </div>
     <header class="header">
         <nav class="navigation">
             <div class="menu-icon">
                 <ul class="mobile-menu">
                     <li class="logo"><a href="">Welcome</a></li>
-                    <li class="menu-btn"><a href=""><img src="img/hamburger-icon.png" alt="Menu Button"></a></li>
+                    <li class="menu-btn" id="menu-btn-1">
+                        <i class="fa-solid fa-bars"></i>
+                    </li>
                 </ul>
                 <ul class="desktop-menu">
-                    <li>Portfolio</li>
-                    <li>About</li>
-                    <li>Contact</li>
+                    <li><a href="#work-section">Portfolio</a></li>
+                    <li><a href="#about-section">About</a></li>
+                    <li><a href="#contact-section">Contact</a></li>
                 </ul>
             </div>
         </nav>
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/201882524-d29f5d64-6d71-4d6b-80c7-478e901d9b13.PNG)

- [ ] You are missing that section in your project menu which has been shown in the Figma image above 👍 Kindly add that section in your mobile menu to make it similar to the Figma template 😊